src/components/content/common/ocl/ContactDetailsText.tsx

```diff
@@ -24,7 +27,7 @@ export function ContactDetailsText({
             <Popover content={<pre>{yamlDocument.toString()}</pre>} title={'Contact Details'} trigger='hover'>
                 <Button className={'ocl-data-hover'} type={'link'}>
                     <ContactsOutlined />
-                    {'support'}
+                    {showFor === ContactDetailsShowType.Order ? ' Contact Service Vendor ' : ' support '}
```

Comment:
we need to remove the fields which have null values. 

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/c7964ad3-12c4-4f80-ae87-56972c6ade90)